### PR TITLE
[ADD] payment_acquirers: add configuration steps for Razorpay

### PR DESCRIPTION
**Pull request by mba-odoo**
task-2800823

See also:
- https://github.com/odoo/odoo/pull/92848

### CODE REVIEW

File: content/applications/finance/payment_acquirers.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers.rst:111]: Checking with a fresh trial database, PayUmoney remains the payment acquirer and I cannot find Razorpay. I guess this is for an upcoming change?

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:19]: Did you mean "Fill the :guilabel:`Secret` **field** with a password...? I feel a word is missing in the sentence.

I would say "Fill the Fill the :guilabel:`Secret` **field** with a password of your choice and save it for later." In my opinion, this seems more clear and more concise.

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:20]: Maybe add a screenshot to make it more clear and understandable.

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:35]: ..important:: has 3 spaces before when it should be flush with the left margin (no spaces)

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:14]: Although the Content Guidelines state 100th character-limit, it is actually a 70-character limit. The sentence should fit within the limit (indicated by the vertical grey line in the screenshot).

70th character

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:38]: These 3 lines also go beyond the 70 char limit.

There's a double '.' at the end of Odoo, I believe there should be only one.

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Hi @mba-odoo ! Great work, but requires a few small modifications before I can approve it. Please have a look :)

File: content/applications/finance/payment_acquirers.rst
Comment (mba-odoo) [reply content/applications/finance/payment_acquirers.rst:119]: @toaa-odoo , Done!

File: content/applications/finance/payment_acquirers.rst
Comment (mba-odoo) [reply content/applications/finance/payment_acquirers.rst:111]: @toaa-odoo , Yes it's for upcoming changes. Thanks

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (toaa-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:21]: This one slipped my attention, it should be :guilabel:`Save` since this a GUI element. My bad! :)

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
All good! I just missed one :guilabel: last time that should be changed.

@jcs-odoo Could you have a look before approval?

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (mba-odoo) [reply content/applications/finance/payment_acquirers/razorpay.rst]: @toaa-odoo , Done..! Thanks

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:6]: "more than 100"

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:18]:
```suggestion
   | For example: `https://example.odoo.com/payment/razorpay/webhook`.
```

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:37]: us guilabel as well if that's how it's displayed on razorpay's website.

Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:37]: avoid the use of future for normal expected behaviour.
Here you could use "would"

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:40]: - numbers up to ten must be written in full
- 
```suggestion
  - After **five days**, if the transaction hasn't been captured yet, the customer has the right
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hello, great addition, thanks.

Here are some additional comments.

Cheers,
Jonathan

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers/razorpay.rst:36]: last missing guilabel haha ;)

**Review by jcs-odoo — APPROVED** [review]

**Review by toaa-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
LGTM from a tech PoV.

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (AntoineVDV) [inline content/applications/finance/payment_acquirers/razorpay.rst:16]: Missing step.
```suggestion
#. | Go to :menuselection:`Settings --> Webhooks`, click on :guilabel:`Create New Webhook`, and enter your Odoo database URL followed by
```

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (AntoineVDV) [inline content/applications/finance/payment_acquirers/razorpay.rst:20]: Swap to follow Razorpay's order.
```suggestion
#. Make sure the :guilabel:`payment.captured` and :guilabel:`refund.processed` checkboxes are ticked.
```

File: content/applications/finance/payment_acquirers/razorpay.rst
Comment (AntoineVDV) [inline content/applications/finance/payment_acquirers/razorpay.rst:21]:
```suggestion
#. Click on :guilabel:`Create Webhook` to finalize the configuration.
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]

**Review by AntoineVDV — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
